test/regression.uts

```diff
@@ -154,40 +171,124 @@ data = ["init_session(\"scapySession2\")",\
 
 emulate_main_input(data)
 
-= Test utility functions
+= Test temporary file creation
+tmpfile = get_temp_file(autoext=".ut")
+if WINDOWS:
+    print tmpfile
+    assert("scapy" in tmpfile and tmpfile.startswith('C:\Users\appveyor\AppData\Local\Temp'))
```

Comment:
You need double '\' ==> assert("scapy" in tmpfile and tmpfile.startswith('C:\\Users\\appveyor\\AppData\\Local\\Temp'))